res/controllers/Numark-Party-Mix.scripts.js

```diff
@@ -23,6 +23,13 @@ NumarkPartyMix.autoLoopSizes = [
 // dim all lights when inactive instead of turning them off
 components.Button.prototype.off = 0x01;
 
+components.Button.prototype.shutdown = function() {
+    if (this.midi === undefined || this.midi[0] === undefined || this.midi[1] === undefined) {
+        return;
+    }
+    midi.sendShortMsg(this.midi[0] - 0x10,this.midi[1] , this.off);
```

Comment:
can you add a comment that explains that the `-0x10` is present to send noteOff instead of NoteOn messages?